Adapters/AdapterBehaviorStandardization/StandardizedActionsAndTriggers.md

```diff
@@ -317,18 +317,23 @@ I know the ID of a customer that I want to delete.
       }
     }
 
-### Make RAW Request
+### Make Raw Request
+A simple action to allow integrators to assemble requests to be sent to the system.  The component should expose the parts that vary in a typical request.  The component should handle authentication and error reporting. It can optionally handle paging. In some cases, it may make sense to allow the option to make a series of sequential array requests.
```

Comment:
If paging is part of the package then the input metadata would require some more optional fields.